src/cdk-experimental/scrolling/virtual-scroll-viewport.scss

```diff
@@ -11,6 +11,7 @@ cdk-virtual-scroll-viewport {
   position: absolute;
   top: 0;
   left: 0;
+  right: 0;
```

Comment:
I think we need to do this differently depending on if we're virtualizing the vertical or horizontal scrolling. For horizontal scrolling we want to set `bottom: 0` instead